src/Plugin.php

```
@@ -0,0 +1,40 @@
+/**
+ * Composer plugin entry point for yii2-extensions/scaffold.
+ *
+ * Registers the event subscriber that orchestrates multi-layer file scaffolding during `composer install`,
+ * `composer update`, and `composer create-project`.
+ *
+ * @author Wilmer Arambula <terabytesoftw@gmail.com>
+ * @since 0.1
+ */
+final class Plugin implements PluginInterface, Capable
+{
+    public function activate(Composer $composer, IOInterface $io): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $io. Configured minimum length is 3. (undefined)

(ShortVariable)